vpr/src/place/place_util.h

```diff
@@ -331,10 +337,11 @@ int get_initial_move_lim(const t_placer_opts& placer_opts, const t_annealing_sch
 double get_std_dev(int n, double sum_x_squared, double av_x);
 
 ///@brief Initialize usage to 0 and blockID to EMPTY_BLOCK_ID for all place_ctx.grid_block locations
-void zero_initialize_grid_blocks();
+void zero_initialize_grid_blocks(GridBlock& grid_blocks);
```

Comment:
comment: place_ctx.grid_block_locations -> grid_blocks
Could make a member function of GridBlock.